static/app.css

```diff
@@ -16,3 +22,29 @@
     background-image: url('https://images.unsplash.com/photo-1597822738124-151fb72dcb79?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1740&q=80');
     background-size: cover;
 }
+
+.find-brewery {
+    position: relative;
+}
+
+#auto {
+    position: absolute;
+    width: 100%;
+    max-height: 30rem;
+    overflow-y: auto;
+}
+
+#auto p {
+    background-color: white;
+    border: solid;
+    border-color: gray;
+    border-width: .5px;
```

Comment:
Please keep consistent with `px` and `rem`

Reply:
all set